### PR TITLE
textarea

### PR DESCRIPTION
**Pull request by alexbeno**

### CODE REVIEW

File: components/atoms/textarea/textarea.core.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+}: TTextareaProps) {
+  const slots = TextareaCoreVariants({ isDisabled: disabled, isInvalid: isError });
+
+  function handleChange(value: string) {
+    onChange?.(value);
+  }
+
+  return (
+    <NextTextarea
+      className="h-fit flex-col items-start gap-2"
```
Comment (haydencleary) [inline components/atoms/textarea/textarea.core.tsx:26]: Is it display flex by default ?

File: components/atoms/textarea/textarea.core.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        helperWrapper: cn(slots.helperWrapper(), classNames?.helperWrapper),
+        description: cn(slots.description(), classNames?.description),
+      }}
+      variant="bordered"
+      labelPlacement="outside-left"
+      isDisabled={disabled}
+      disabled={disabled}
+      onValueChange={handleChange}
+      value={value}
+      multiple={multiple}
```
Comment (haydencleary) [inline components/atoms/textarea/textarea.core.tsx:44]: What is this for ?

File: components/atoms/textarea/textarea.variants.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import { tv } from "tailwind-variants";
+
+export const TextareaCoreVariants = tv({
+  slots: {
+    base: "h-fit flex-col items-start gap-2",
```
Comment (haydencleary) [inline components/atoms/textarea/textarea.variants.ts:5]: Display flex ?

File: components/atoms/textarea/textarea.core.tsx (hunk 1 above)
Comment (haydencleary) [reply components/atoms/textarea/textarea.core.tsx:26]: And isn't this applied in the classNames base ?

**Review by haydencleary — CHANGES_REQUESTED** [review]

File: components/atoms/textarea/textarea.core.tsx (hunk 2 above)
Comment (alexbeno) [reply components/atoms/textarea/textarea.core.tsx:44]: My bad typed error, I will extends minRows, maxRows and disableAutosize

**Review by haydencleary — APPROVED** [review]

**Pull request merged**